src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1027,7 +1102,16 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 				$body[] = __( 'These themes failed to update:' );
 
 				foreach ( $failed_updates['theme'] as $item ) {
-					$body[] = "- {$item->name}";
+					$body[]                               = "- {$item->name}";
+					
+					// reset the plugin time if time has been passed
+					if ( isset($failure_emails[ $item->item->theme ] ) ) {
+						if ( time() - $failure_emails[ $item->item->theme ] > $failure_email_interval ) {
+							$failure_emails[ $item->item->theme ] = time();
+						}
+					} else {
+						$failure_emails[ $item->item->theme ] = time();
+					}
```

Comment:
Reset the plugin or theme time if time has been passed for 3 days or more

Reply:
This logic is not needed. The one line that was there was sufficient.